client/src/pages/Payment.js

```diff
@@ -0,0 +1,298 @@
+import React, { useContext, useEffect, useState } from "react";
+import NavBar from "../components/NavBar";
+import { makeStyles } from "@material-ui/core/styles";
+import { Typography, Button } from "@material-ui/core";
+// stripe
+import { CardElement, useStripe, useElements } from "@stripe/react-stripe-js";
+import { UserContext } from "../context/UserContext";
+import Loader from "../components/Loader";
+import sushi from "../assets/sushi.png";
+
+export const useStyles = makeStyles((theme) => ({
+  root: {
+    height: "100vh",
+    scrollY: "hidden",
+  },
+  paper: {
+    margin: theme.spacing(3, 8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+  },
+  margin: {
+    margin: theme.spacing(4, 4),
+  },
+
+  form: {
+    width: "30vw",
+    alignSelf: "center",
+
+    borderRadius: "7px",
+    padding: "40px",
+  },
+
+  input: {
+    borderRadius: "6px",
+    marginBottom: "6px",
+    padding: "12px",
+    border: "1px solid rgba(50, 50, 93, 0.1)",
+    maxHeight: "44px",
+    fontSize: "16px",
+    width: "100%",
+    background: "white",
+    boxSizing: "border-box",
+  },
+  container: {
+    display: "flex",
+  },
+
+  left__container: {
+    display: "flex",
+    flexDirection: "column",
+    flex: "0.6",
+    height: "90vh",
+    background: "white",
+    overflowY: "hidden",
+  },
+  right__container: {
+    display: "flex",
+    flexDirection: "column",
+    flex: "0.4",
+  },
+  logo: {
+    margin: "20px",
+    padding: "20px",
+    borderBottom: "1px solid lightgrey",
+  },
+
+  order: {
+    marginTop: "50px",
+    marginLeft: "50px",
+    marginRight: "50px",
+    background: "white",
+    padding: "20px",
+
+    "& h4": {
+      textAlign: "center",
+    },
+  },
+
+  orderDetails: {
+    display: "flex",
+    "& p": {
+      marginTop: "30px",
+      textAlign: "left",
+    },
+  },
+
+  btn: {
+    borderRadius: "0",
+    height: theme.spacing(7),
+    margin: "0 50px",
+    paddingLeft: "20px",
+    paddingRight: "20px",
+    textTransform: "capitalize",
+    fontWeight: "500",
+  },
+  cardError: {
+    margin: "20px",
+    color: "rgb(105, 115, 134)",
+    fontSize: "16px",
+    lineHeight: "20px",
+    marginTop: "12px",
+    textAlign: "center",
+  },
+
+  img: {
+    height: "100px",
+    width: "100px",
+    margin: "20px 20px",
+  },
+
+  orderTotal: {
+    color: "#FF743D",
+    fontWeight: "bold",
+  },
+
+  paymentDetails: {
+    marginLeft: "12px ",
+    marginBottom: "20px ",
+  },
+
+  title: {
+    paddingLeft: "60px",
+  },
+
+  resultMessage: {
+    lineHeight: "22px",
+    fontSize: "16px",
+  },
+
+  hidden: {
+    display: "none",
+  },
+}));
+
+const cardStyle = {
+  style: {
+    base: {
+      color: "#32325d",
+      fontFamily: "Arial, sans-serif",
+      fontSmoothing: "antialiased",
+      fontSize: "16px",
+      "::placeholder": {
+        color: "#32325d",
+      },
+    },
+    invalid: {
+      color: "#fa755a",
+      iconColor: "#fa755a",

```

Comment:
Might want to define some colors in your theme.js so you can call theme.palette instead of hex/rgb values